crates/ark/src/lsp/completions/sources/composite/keyword.rs

```diff
@@ -63,11 +115,100 @@ pub fn completions_from_keywords() -> anyhow::Result<Option<Vec<CompletionItem>>
             continue;
         });
 
-        item.detail = Some("[keyword]".to_string());
         item.kind = Some(CompletionItemKind::KEYWORD);
+        item.label_details = Some(CompletionItemLabelDetails {
+            detail: None,
+            description: Some("[keyword]".to_string()),
+        });
```

Comment:
The keyword completion items were actually using the fields of `CompletionItem` differently than the other sources, leading to undesirable behaviour in the UI. This `[keyword]` wouldn't show up by default, the way that, say, `{methods}` would. Instead it would only show up if that item was queued for selection. Here's how things look *with* this change:
<img width="466" alt="Screenshot 2025-04-24 at 7 17 09 PM" src="https://github.com/user-attachments/assets/1ded26ae-1461-4dee-951a-ef7a6c1267b5" />